### PR TITLE
Change Three v79 to v80

### PR DESCRIPTION
**Pull request by gchoqueux**
the Three library now uses ES6 Modules

### CODE REVIEW

File: src/Renderer/ThreeExtented/GlobeControls.js [hunk 1]
```diff
@@ -225,8 +224,9 @@ function GlobeControls(camera, domElement, engine) {
 
 
     if(enableTargetHelper)
-
+    {
```
Comment (peppsac) [inline src/Renderer/ThreeExtented/GlobeControls.js:232]:
``` js
if (enableTargetHelper) {
```

File: src/Renderer/ThreeExtented/GlobeControls.js [hunk 2]
```diff
@@ -445,7 +445,9 @@ function GlobeControls(camera, domElement, engine) {
             var delta = coord.altitude()  - (bbox.top() + radiusCollision);
 
             if(delta<0)
+            {
```
Comment (peppsac) [inline src/Renderer/ThreeExtented/GlobeControls.js]:
``` js
if (delta < 0) {
```

**Review by peppsac — COMMENTED** [review]
I'm not sure that using `import * as THREE from 'THREE';` is a good idea.
I'd prefer to explicitely list imported elements ; what do you think?

**Comment by gchoqueux** [comment]
> I'm not sure that using import \* as THREE from 'THREE'; is a good idea.

Yes, I think too `import * as THREE from 'THREE';` is not a best idea.

For example in https://github.com/iTowns/itowns2/blob/ThreeV80/src/Renderer/BasicMaterial.js#L8

``` js
import * as THREE from 'THREE';
```

becomes

``` js
import {RawShaderMaterial,Color,Matrix4} from 'THREE';
```

But we lose name's module

``` js
THREE.RawShaderMaterial.call(this);
```

becomes

``` js
RawShaderMaterial.call(this);
```

I didn't find import method to load specific member and keeping name's module :(
I think, it's better to keep name's module `THREE`

**Comment by tbroyer** [comment]
What's wrong with `import * as THREE from 'three'`? That's exactly what the [pull request making the change to Three.js](https://github.com/mrdoob/three.js/pull/9310) suggests.

**Comment by peppsac** [comment]
In most language wildcard imports are considered a bad practice, hence my question.
In Javascript these imports are scoped thanks to the `as XXX` syntax so I suppose that's not an issue.

So it looks good but I need to test before giving my approval :-)

**Comment by peppsac** [comment]
LGTM

**Pull request merged**